docs/fulltext.adoc

```diff
@@ -146,7 +146,7 @@ All https://neo4j.com/docs/java-reference/3.1/#indexing-create-advanced[standard
 | name | value | description
 | type | fulltext/exact | type of the index
 | to_lower_case | false/true | if terms should be converted to lower case before indexing
-| analyzer | classname | classname of lucene analyzer to be used for this index
+| analyzers | classnames | list of classname of lucene analyzer to be used for this index
```

Comment:
why the change in the config name?

Reply:
It's mistake. The right string is "anayzer" but you must pay attention that analizer is array now